src/lib/arrays.js

```diff
@@ -0,0 +1,104 @@
+const getNthElement = (index, array) => {
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join();
+};
+
+const csvStringToArray = string => {
+  return string.split(',', 5);
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  const newArray = array.concat(element);
+  return newArray;
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = numbers => {
+  const numtoString = numbers.map(num => {
+    return String(num);
+  });
+  return numtoString;
```

Comment:
Theres no need to create the function here. You can just return the value of the map function directly without the additional call:
```
  return numbers.map(num => {
    return String(num);
  });
```